CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the "aws-vscode-tools" extension will be documented in this file.
 
+## NEXT (Developer Preview)
```

Comment:
have you seen the JSON versions of the change log that jetbrains / java sdk / python sdk etc use?
https://github.com/aws/aws-toolkit-jetbrains/blob/master/.changes/next-release/feature-28e0ea34-b749-4750-8ee2-bd8c92be78df.json

Having these programmatically consumable means you can do things like group all features together - and you get fewer commit conflicts when two people edit the changelog file.

Reply:
We have issue #381 to onboard a changelog system.